app/mailers/invite_mailer.rb

```diff
@@ -0,0 +1,13 @@
+class InviteMailer < Devise::Mailer
+
+  protected
```

Comment:
This is protected, so I figured there was no need to add a spec for it (let me know if that's a wrong assumption at all.)